crates/re_log_types/src/lib.rs

```diff
@@ -205,6 +205,27 @@ impl std::fmt::Display for ApplicationId {
 
 // ----------------------------------------------------------------------------
 
+/// Options for configuring viewer behavior when a blueprint is ready.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)] // `PartialEq` used for tests in another crate
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub struct BlueprintReadyOpts {
+    /// Make this the active blueprint for the `app_id`.
+    pub make_active: bool,
+
+    /// Make this the default blueprint for the `app_id`.
+    pub make_default: bool,
```

Comment:
Are all 2x2 combinations of these a valid and unique command?

Reply:
In theory yes, but false/false is maybe worth warning about.